06-other-features.R

```diff
@@ -10,7 +10,7 @@ urls <- list(
 
 map(urls, read_lines)
 
-safe_readLines <- safely(readLines)
+safe_readlines <- safely(readLines)
```

Comment:
Good catch, can you also change  line #17 to use `safe_readlines`? Then I'm happy to merge